src/playground/components/Configuration.js

```diff
@@ -208,6 +233,7 @@ export default function Configuration({ rulesMeta, eslintVersion, onUpdate, opti
                     <label htmlFor="rules" className="combo-label"><span className="label__text">Choose a rule</span></label>
                     <Select
                         isMulti
+                        components={{ Input }}
```

Comment:
```closeOnMultiSelect={false}```
This could be added to all multi select components. since everytime a we choose a rule, the dropdown closes due to closeOnMultiSelect is set to true as default

Reply:
We have to confirm if this is expected or not. I think this is expected?